File: src/main/java/net/gtaun/shoebill/SampObjectManager.java
package net.gtaun.shoebill;

public interface SampObjectManager extends SampObjectStore, SampObjectFactory
{
    /**
     * Get the SampObjectManager.
     * @return SampObjectManger.
     */
	public static SampObjectManager get()
	{
		return Shoebill.get().getSampObjectManager();
	}
}

File: src/main/java/net/gtaun/shoebill/SampObjectManager.kt
package net.gtaun.shoebill

interface SampObjectManager : SampObjectStore, SampObjectFactory {
    companion object {
        /**
         * Gets the [SampObjectManager].
         */
        fun get(): SampObjectManager {
            return Shoebill.get()!!.sampObjectManager
        }
    }
}
